cmd/minikube/cmd/start.go

```diff
@@ -1060,6 +1064,35 @@ func validateRegistryMirror() {
 	}
 }
 
+// This function validates that the --insecure-registry follows one of the following formats:
+// "<ip>:<port>" "<hostname>:<port>" "<network>/<netmask>"
+func ValidateInsecureRegistry() {
```

Comment:
this should be a private function -- and probably moved to start_flags.go.

Reply:
Agree with the private function, about the location I've placed it under the "validateRegistryMirror" function since they do pretty much the same. To me it makes sense having both together, what do you think?